src/common/util/src/file_util.cpp

```diff
@@ -634,3 +634,18 @@ void ov::util::save_binary(const std::string& path, std::vector<uint8_t> binary)
         throw std::runtime_error("Could not save binary to " + path);
     }
 }
+
+void ov::util::save_binary(const std::string& path, const char* binary, size_t bin_size) {
```

Comment:
can we reimplement `void ov::util::save_binary(const std::string& path, std::vector<uint8_t> binary)` via this new method to avoid code duplication?

Reply:
Ok.